scripter/scripter.go

```diff
@@ -154,3 +164,14 @@ func SetBasicMethods(c ScrConn, service string) {
 		}
 	}, service)
 }
+
+// ReloadScripts reloads the scripts from the scripter
+func ReloadScripts(s Scripter) {
+	for service := range s.GetScripts() {
+		if err := s.Init(service); err != nil {
+			log.Errorf("error init service: %s", err)
```

Comment:
This error message is confusing, I would make it: Error on reload, init on service failed: